File: Python3/FizzBuzz.py
def main():
    FizzBuzz(100)

def FizzBuzz(max):
    for i in range(1,max):
        oStr = ""
        if i%3==0:
            oStr = "Fizz"

        if i%5==0:
            if oStr!="":
                oStr += " "
            oStr += "Buzz"
        
        if oStr=="":
            oStr=i

        print(oStr)

if __name__ == "__main__":
    main()

File: R/FizzBuzz.r
max <- 100

for(i in 1:max){
    output <- ""
    if(i%%3==0)
        output <- "Fizz"

    if(i%%5==0)
        if(output != "")
            output <- paste(output, "Buzz")
        else
            output <- paste0(output, "Buzz")

    if(output == "")
        output <- i

    print(output, quote = FALSE)
}
